src/components/slider/slider.spec.js

```diff
@@ -23,13 +23,30 @@ describe('md-slider', function() {
       'getBoundingClientRect'
     ).and.returnValue(angular.extend({
       width: 100,
+      height: 100,
       left: 0,
-      right: 0
+      right: 0,
+      bottom: 0,
+      top: 0
     }, dimensions || {}));
 
     return slider;
   }
 
+  function setupContainer(attrs, sliderAttrs) {
+    var container;
+
+    container = $compile('<md-slider-container ' + (attrs || '') + '>' +
```

Comment:
Can you refactor the HTML here into a `template` variable? With the embedded parentheses, it makes it a bit hard to read :smile: